package.json

```diff
@@ -17,7 +17,7 @@
     "lint-staged": "lint-staged",
     "format:fix": "prettier --write \"**/*.{ts,tsx,json,scss,css}\"",
     "format:check": "prettier --check \"**/*.{ts,tsx,json,scss,css}\"",
-    "prepare": "husky",
+    "prepare": "husky init",
```

Comment:
Why change this? Husky wasn't updated in the package list.

Reply:
No, it is a merge of my PR 

- #2018

Reply:
This PR changes has been removed through this PR merge 

- #2046

Reply:
Even this PR made a lot of breaking changes.
```
  const query = {
    userId: currentUser._id.toString(),
    organizationId: organization._id.toString(),
```

It is the reason userid isn't querying and made a issue fo it